<launch>
    <param name="robot_description"
    command="$(find xacro)/xacro --inorder '$(find first_rrr)/urdf/robot_rrr.xacro'" />
    <node name="kinematic_simulator" pkg="kinematic_simulator" type="kin_sim" output="screen"/>

  <!--  <node name="joint_state_publisher" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui"/>
-->
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

     <node name="rviz" pkg="rviz" type="rviz" args="-d $(find first_rrr)/urdf/conf.rviz"/> 

</launch>